### PR TITLE
Export single INTERLIS file

### PR DESCRIPTION
**Pull request by Alelizzt**

### CODE REVIEW

File: src/ch/ehi/umleditor/application/NavigationView.java [hunk 1]
```diff
@@ -664,6 +667,34 @@ private void connEtoC29(java.awt.event.ActionEvent arg1) {
...
+		// user code begin {2}
+		// user code end
+	} catch (java.lang.Throwable ivjExc) {
+		// user code begin {3}
+		// user code end
+		handleException(ivjExc);
+	}
+}
+	private void mniIliExport(INTERLIS2Def exportili) {
+	// TODO Auto-generated method stub
```
Comment (claeis) [inline src/ch/ehi/umleditor/application/NavigationView.java]: pls remove this outdated comment

File: src/ch/ehi/umleditor/application/NavigationView.java [hunk 2]
```diff
@@ -1358,6 +1389,29 @@ private void connEtoC9(java.awt.event.ActionEvent arg1) {
...
+ */
+/* WARNING: THIS METHOD WILL BE REGENERATED. */
+private javax.swing.JMenuItem getMniExportIli() {
+	if (ivjMniExportIli == null) {
+		try {
+			ivjMniExportIli = new javax.swing.JMenuItem();
+			ivjMniExportIli.setName("MniExportIli");
+			ivjMniExportIli.setText("Export");
+			// user code begin {1}
+			ivjMniExportIli.setText(getResourceString("MniExportIli_text"));
```
Comment (claeis) [inline src/ch/ehi/umleditor/application/NavigationView.java]: where is this resource

File: src/ch/ehi/umleditor/interlis/iliexport/ExportInterlis.java [hunk 3]
```diff
@@ -44,7 +46,17 @@ public static void writeFileset()
...
     }
-
+  
+  public static void writeIli(INTERLIS2Def ili) {
+		TransferFromUmlMetamodel writer = new TransferFromUmlMetamodel();
+		try {
+			writer.getFileList(ili.getNamespace());
+			writer.writeIliFile(ili);
+		}catch (java.io.IOException ex) {
+			ch.ehi.umleditor.application.LauncherView.getInstance().log("Export interlis file", "Fail when try write the interlis file");
```
Comment (claeis) [inline src/ch/ehi/umleditor/interlis/iliexport/ExportInterlis.java]: pls use ex.getLocalizedMessage() (so the user sees, what went wrong)

Comment (Alelizzt) [reply src/ch/ehi/umleditor/interlis/iliexport/ExportInterlis.java]: something like this?
ch.ehi.umleditor.application.LauncherView.getInstance().log("Export interlis file", ex.getLocalizedMessage());

File: src/ch/ehi/umleditor/application/NavigationView.java (hunk 2 above)
Comment (Alelizzt) [reply src/ch/ehi/umleditor/application/NavigationView.java]: I will add it

File: src/ch/ehi/umleditor/application/NavigationView.java (hunk 1 above)
Comment (Alelizzt) [reply src/ch/ehi/umleditor/application/NavigationView.java]: ok

File: src/ch/ehi/umleditor/interlis/iliexport/ExportInterlis.java (hunk 3 above)
Comment (claeis) [reply src/ch/ehi/umleditor/interlis/iliexport/ExportInterlis.java]: yes

**Pull request merged**